Documentation/gettingstarted/aws-eni.rst

```diff
@@ -68,6 +68,62 @@ Deploy Cilium release via Helm:
 .. include:: namespace-kube-system.rst
 .. include:: hubble-enable.rst
 
+ENI Subnet tags
+===============
+
+To allow the Cilium Operator to filter subnets by tag in ENI mode, there are
+two pieces of configuration that must be provided, a custom CNI configuration
+and a field in the Cilium ``ConfigMap``:
+
+Create a CNI configuration
+--------------------------
+
+Create a `cni-config.yaml` file based on the template below. Fill in the
+``subnet-tags`` field:
+
+.. code:: yaml
+
+   apiVersion: v1
+   kind: ConfigMap
+   metadata:
+     name: cni-configuration
+     namespace: kube-system
+   data:
+     cni-config: |-
+       {
+         "cniVersion":"0.3.1",
+         "name":"cilium",
+         "plugins": [
+           {
+             "cniVersion":"0.3.1",
+             "type":"cilium-cni",
+             "eni": {
+               "subnet-tags":{
+                 "foo":"true"
```

Comment:
I guess there's an implicit assumption here that the user has configured these specific subnet-tags already in some AWS system?

Reply:
Yes, that's true. I'll add a note of that above.